<ui version="4.0" >
 <class>ForgotPasswordWidgetClass</class>
 <widget class="QWidget" name="ForgotPasswordWidgetClass" >
  <property name="windowModality" >
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>267</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <widget class="QFrame" name="frame_2" >
   <property name="geometry" >
    <rect>
     <x>9</x>
     <y>9</y>
     <width>370</width>
     <height>249</height>
    </rect>
   </property>
   <property name="frameShape" >
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth" >
    <number>0</number>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2" >
    <property name="spacing" >
     <number>6</number>
    </property>
    <property name="margin" >
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="topTitleFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="titleLabel" >
         <property name="accessibleName" >
          <string>redLabel</string>
         </property>
         <property name="text" >
          <string>Title</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3" >
       <property name="leftMargin" >
        <number>40</number>
       </property>
       <property name="rightMargin" >
        <number>40</number>
       </property>
       <item>
        <spacer name="verticalSpacer_4" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="loginLabel" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="accessibleName" >
          <string>whiteLabel</string>
         </property>
         <property name="text" >
          <string>Login</string>
         </property>
         <property name="alignment" >
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="buddy" >
          <cstring>loginEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="loginEdit" />
       </item>
       <item>
        <spacer name="verticalSpacer_5" >
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>10</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="bottomLinksFrame" >
      <property name="sizePolicy" >
       <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize" >
       <size>
        <width>16777215</width>
        <height>32</height>
       </size>
      </property>
      <property name="frameShape" >
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <property name="lineWidth" >
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <property name="spacing" >
        <number>30</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <item>
        <spacer name="horizontalSpacer_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>10</width>
           <height>14</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="backButton" >
         <property name="font" >
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>true</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor" >
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="accessibleName" >
          <string>blueButton</string>
         </property>
         <property name="text" >
          <string>Back</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="nextButton" >
         <property name="font" >
          <font>
           <family>MS Shell Dlg 2</family>
           <pointsize>8</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
           <underline>true</underline>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="cursor" >
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="accessibleName" >
          <string>blueButton</string>
         </property>
         <property name="text" >
          <string>Next</string>
         </property>
         <property name="flat" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>10</width>
           <height>14</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
   <zorder>frame</zorder>
   <zorder>bottomLinksFrame</zorder>
   <zorder>topTitleFrame</zorder>
  </widget>
  <widget class="QFrame" name="transBorder_4" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>2</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder" >
   <property name="geometry" >
    <rect>
     <x>386</x>
     <y>2</y>
     <width>2</width>
     <height>263</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth" >
    <number>1</number>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder_2" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>2</y>
     <width>2</width>
     <height>263</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>2</width>
     <height>263</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
  <widget class="QFrame" name="transBorder_3" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>265</y>
     <width>401</width>
     <height>2</height>
    </rect>
   </property>
   <property name="minimumSize" >
    <size>
     <width>0</width>
     <height>2</height>
    </size>
   </property>
   <property name="maximumSize" >
    <size>
     <width>16777215</width>
     <height>2</height>
    </size>
   </property>
   <property name="accessibleName" >
    <string>transBorder</string>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Plain</enum>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources>
  <include location="moodbox.qrc" />
 </resources>
 <connections/>
</ui>
